ci/cudf_pandas_scripts/pandas-tests/run.sh

```diff
@@ -11,8 +11,11 @@ rapids-logger "Running Pandas tests using $PANDAS_TESTS_BRANCH branch and rapids
 rapids-logger "PR number: ${RAPIDS_REF_NAME:-"unknown"}"
 
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
-RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./local-cudf-dep
-python -m pip install $(ls ./local-cudf-dep/cudf*.whl)[test,pandas-tests]
+RAPIDS_PY_WHEEL_NAME="cudf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
```

Comment:
I don't care whether the local directory used for this purpose is called `./dist` or `./local-cudf-dep`. Happy to defer to whatever reviewers want.

But having it be different across and within these testing scripts was a little bit of friction I think we could avoid. I don't think there's any functional reason to use those two different directory names in these testing scripts, please correct me if I'm wrong.